packages/react-native-client/android/src/main/java/io/fishjam/reactnative/RNFishjamClient.kt

```diff
@@ -263,7 +263,9 @@ class RNFishjamClient(
     isScreenShareOn = false
     isConnected = false
     isCameraInitialized = false
-    fishjamClient.leave()
+    fishjamClient.leave {
```

Comment:
**question:** Why is the callback needed? Looks like this should be synchronous 🤔

Reply:
On Android everything in `leave` is called in `coroutineScope.launch` so it's asynchronous. I believe that there is a reason why it was added so I didn't want to change that.
On iOS it is synchronous but I wanted it to match its Android counterpart.